### ISSUE TITLE
LEETCODE: PROBLEMS-

### ISSUE THREAD

**Issue by JackDan9** [jdneo/vscode-leetcode#190]
node-version in the Windows env:
![image](https://user-images.githubusercontent.com/12805846/53882062-3eff7080-4050-11e9-99fe-3922bb19bedf.png)
VSCode info about my version:
```
Version: 1.31.1 (user setup)
Commit: 1b8e8302e405050205e69b59abb3559592bb9e60
Date: 2019-02-12T02:20:54.427Z
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134
```
Question Picture Like this:
![image](https://user-images.githubusercontent.com/12805846/53882144-78d07700-4050-11e9-8735-327dbe6652ec.png)

Please give me a hand. Thanks!

## 💬 Questions and Help

Please outline the questions.

**Comment by jdneo** [comment]
Wired. Does your VS Code started from the command line that has Node.js in its env?

**Comment by JackDan9** [comment]
Yes, Do you mean the picture below?
```
C:\Users\JackDan9>Code

C:\Users\JackDan9>node -v
v10.15.0

C:\Users\JackDan9>
```
Thanks

**Comment by jdneo** [comment]
@JackDan9 Hmm...

Could you see any output in the LeetCode output Channel?

Or, is there any error message? You can check this by Help > Toggle Developer Tools > to see if there is anything in the console.

**Comment by JackDan9** [comment]
Ok，I saw the console information of the error message, like the follow picture.
![image](https://user-images.githubusercontent.com/12805846/54024641-5024ba80-41d3-11e9-84b3-d9b58bf3c19c.png)
Thanks.

**Comment by jdneo** [comment]
What if you remove the folder `$HOME/.lc` and try again?

**Comment by jdneo** [comment]
@JackDan9 Another possibility is that your network environment prevents you to download the plugins sources from `Github`

**Comment by JackDan9** [comment]
I deleted this directory `$Home/.lc` file and closed the firewall. @jdneo But have the same question. 
Thanks.

**Comment by jdneo** [comment]
Suspecting releated with #211

**Comment by jdneo** [comment]
I'm closing this issue for now since this issue should be fixed with #212.

We will release the fix recently in `0.13.1`.

If you still have problems after using `0.13.1`, please feel free to let us know.
